tests/core/eth-module/test_gas_pricing.py

```diff
@@ -14,14 +14,14 @@ def test_get_set_gasPrice(web3):
 
 
 def test_no_gas_price_strategy_returns_none(web3):
```

Comment:
Is this the one we paired on trying to add a deprecation warning? If not, I'd expect a test that ensures the `DeprecationWarning` gets raised somewhere in this file.